### PR TITLE
Image & View: add snack examples to use functional components

### PR DESCRIPTION
**Pull request by amandeepmittal**
In Reference to #1579 

This PR adds a Snack Example for `<Image/>` component with a use case for a functional component.

### CODE REVIEW

**Comment by Simek** [comment]
Few semicolons are missing mainly after `export` statements and I would prefer arrow functions over `function` keyword but other than that LGTM.

**Comment by rachelnabors** [comment]
@amandeepmittal can you make the changes @Simek suggested?

**Comment by amandeepmittal** [comment]
Oh, my apologies to both of you @rachelnabors and @Simek but I the notifications. 🙏
I will update it with the changes.

**Comment by amandeepmittal** [comment]
Hey @Simek I have made changes as you suggested. Couldn't find any missing semi colons, also, I am using the same prettier config as the repo states.

File: docs/image.md [hunk 1]
```diff
@@ -9,35 +9,162 @@ This example shows fetching and displaying an image from local storage as well a
...
         />
         <Image
-          style={{width: 66, height: 58}}
+          style={styles.logo}
           source={{uri: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADMAAAAzCAYAAAA6oTAqAAAAEXRFWHRTb2Z0d2FyZQBwbmdjcnVzaEB1SfMAAABQSURBVGje7dSxCQBACARB+2/ab8BEeQNhFi6WSYzYLYudDQYGBgYGBgYGBgYGBgYGBgZmcvDqYGBgmhivGQYGBgYGBgYGBgYGBgYGBgbmQw+P/eMrC5UTVAAAAABJRU5ErkJggg=='}}
         />
       </View>
     );
   }
 }
```
Comment (Simek) [inline docs/image.md]: Missing semicolon 😉

File: docs/image.md [hunk 2]
```diff
@@ -61,8 +188,14 @@ export default class DisplayAnImageWithStyle extends Component {
...
```
Comment (Simek) [inline docs/image.md]: Missing semicolon 😉

File: docs/view.md [hunk 3]
```diff
@@ -9,23 +9,26 @@ The most fundamental component for building a UI, `View` is a container that sup
...
+        flexDirection: 'row',
+        height: 100,
+        padding: 20,
+      }}>
+      <View style={{ backgroundColor: 'blue', flex: 0.3 }} />
+      <View style={{ backgroundColor: 'red', flex: 0.5 }} />
+      <Text>Hello World!</Text>
+    </View>
+  );
 }
```
Comment (Simek) [inline docs/view.md]: Missing semicolon 😉

File: docs/view.md [hunk 4]
```diff
@@ -9,23 +9,26 @@ The most fundamental component for building a UI, `View` is a container that sup
...
-        <View style={{backgroundColor: 'red', flex: 0.5}} />
-        <Text>Hello World!</Text>
-      </View>
-    );
-  }
+```SnackPlayer name=Example%20Code
+import React from 'react';
+import { View, Text } from 'react-native';
+
+const ViewBoxesWithColorAndText= () => {
```
Comment (Simek) [inline docs/view.md]: Missing space 😉

**Comment by amandeepmittal** [comment]
Sorry about that @Simek I thought the prettier config of the repo was handling it. 

Thank you @rachelnabors for this opportunity.

**Comment by Simek** [comment]
@amandeepmittal No problem, thank you for the contribution! 😄

**Pull request merged**